### PR TITLE
Unify schema into modeling rules

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

File: demisto_sdk/commands/unify/rule_unifier.py [hunk 1]
```diff
@@ -61,3 +62,10 @@ def _insert_samples(self):
                 click.echo(f'Added {len(samples)} samples.')
             else:
                 click.echo('Did not find matching samples.')
+
+    def _insert_schema(self):
+        schema_path = Path(self.yml_path).with_suffix('.json')
+        if os.path.exists(schema_path):
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/unify/rule_unifier.py:68]: If we didn't find any schema let's print to logs that no schema was found.

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,7 @@
 * Removed a traceback line from the **init** command templates: BaseIntegration, BaseScript.
 * Updated the token in **_add_pr_comment** method from the content-bot token to the xsoar-bot token.
 * Added a new validation to the **validate** command to verify that the docker in use is not deprecated.
+* Added support in the **unify** command to unify the schema into the Modeling Rule.
```
Comment (ShahafBenYakir) [inline CHANGELOG.md:27]:
```suggestion
* Added support in the **unify** command to unify a schema into its Modeling Rule.
```

**Review by ShahafBenYakir — COMMENTED** [review]
Cool
1. Unit Tests?
2. Need to open a PR in Content to move all schemas outside the yml into JSON files and make sure your unify feature work. We will merge the content PR only after the next release of the SDK version.

**Comment by moishce** [comment]
- [PR in Content to move all schemas outside the yml into JSON files](https://github.com/demisto/content/pull/19630)
- a [successful upload and unify](https://code.pan.run/xsoar/content/-/pipelines/3079375) with the unify in a [test branch](https://github.com/demisto/content/pull/19461)

**Comment by ShahafBenYakir** [comment]
Please also see this https://github.com/demisto/content/pull/19461#issuecomment-1160217142

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,7 @@
 * Added a **validate** check, making sure classifier id and name values match. Updated the classifier **format** to update the id accordingly.
 * The **generate-docs** command will now auto-generate the playbook image link as default. Added the `--custom-image-link' argument to override.
```
Comment (ShahafBenYakir) [inline CHANGELOG.md:11]: There's a bug here, can you fix it also?
```suggestion
* The **generate-docs** command will now auto-generate the playbook image link by default. 
* Added the `--custom-image-link` argument to override.
```

**Review by ShahafBenYakir — APPROVED** [review]

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**